apps/passport-server/src/database/queries/pipelineDefinitionDB.ts

```diff
@@ -234,7 +251,51 @@ export class PipelineDefinitionDB implements IPipelineDefinitionDB {
     definitions: PipelineDefinition[]
   ): Promise<void> {
     for (const definition of definitions) {
-      await this.upsertDefinition(definition);
+      await this.upsertDefinition(definition, undefined);
     }
   }
+
+  public async appendToEditHistory(
+    pipelineDefinition: PipelineDefinition,
+    editorUserId?: string
+  ): Promise<void> {
+    await this.db.query(
+      `
+    insert into podbox_edit_history
+    (pipeline, editor_user_id, time_created)
+    values
+    ($1, $2, $3);`,
+      [pipelineDefinition, editorUserId, new Date()]
```

Comment:
I'm assuming that if `editorUserId` is undefined then the field ends up being `NULL` in the DB?